mockwebserver/src/main/java/okhttp3/mockwebserver/RecordedRequest.kt

```diff
@@ -30,40 +30,50 @@ import javax.net.ssl.SSLSocket
 
 /** An HTTP request that came into the mock web server. */
 class RecordedRequest(
-  val requestLine: String,
+  @get:JvmName("requestLine") val requestLine: String,
+
   /** All headers. */
-  val headers: Headers,
+  @get:JvmName("headers") val headers: Headers,
+
   /**
    * The sizes of the chunks of this request's body, or an empty list if the request's body
    * was empty or unchunked.
    */
-  val chunkSizes: List<Int>,
+  @get:JvmName("chunkSizes") val chunkSizes: List<Int>,
+
   /** The total size of the body of this POST request (before truncation).*/
-  val bodySize: Long,
+  @get:JvmName("bodySize") val bodySize: Long,
+
   /** The body of this POST request. This may be truncated. */
-  val body: Buffer,
+  @get:JvmName("body") val body: Buffer,
+
   /**
    * The index of this request on its HTTP connection. Since a single HTTP connection may
    * serve multiple requests, each request is assigned its own sequence number.
    */
-  val sequenceNumber: Int,
-  socket: Socket
+  @get:JvmName("sequenceNumber") val sequenceNumber: Int,
+
+  internal val socket: Socket
 ) {
-  val method: String?
-  val path: String?
+  @get:JvmName("method") val method: String?
+  @get:JvmName("path") val path: String?
+
   /**
    * The TLS handshake of the connection that carried this request, or null if the request
    * was received without TLS.
    */
-  val handshake: Handshake?
-  val requestUrl: HttpUrl?
+  @get:JvmName("handshake") val handshake: Handshake?
+  @get:JvmName("requestUrl") val requestUrl: HttpUrl?
 
   val utf8Body: String
```

Comment:
We should replace this `val` with a `fun`; it doesn’t behave like a `val` in that reading it has a side-effect